mm2src/coins/qrc20/rpc_electrum.rs

```diff
@@ -0,0 +1,142 @@
+use super::*;
+
+#[derive(Debug, Deserialize, Eq, PartialEq)]
+pub struct TokenInfo {
+    name: String,
+    decimals: u8,
+    total_supply: u64,
+    symbol: String,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct ExecutionResult {
+    pub output: BytesJson,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct ContractCallResult {
+    address: H160Json,
+    #[serde(rename = "executionResult")]
+    pub execution_result: ExecutionResult,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct TxHistoryItem {
+    pub tx_hash: H256Json,
+    pub height: i64,
+    pub log_index: i64,
+}
+
+/// The structure is the same as Qtum Core RPC gettransactionreceipt returned data.
+/// https://docs.qtum.site/en/Qtum-RPC-API/#gettransactionreceipt
+#[derive(Debug, Deserialize)]
+pub struct TxReceipt {
+    /// Hash of the block this transaction was included within.
+    #[serde(rename = "blockHash")]
+    pub block_hash: H256Json,
+    /// Number of the block this transaction was included within.
+    #[serde(rename = "blockNumber")]
+    pub block_number: i64,
+    /// Transaction hash.
+    #[serde(rename = "transactionHash")]
+    pub transaction_hash: H256Json,
+    /// Index within the block.
+    #[serde(rename = "transactionIndex")]
+    pub transaction_index: i64,
+    /// Index within the outputs.
+    #[serde(rename = "outputIndex")]
+    pub output_index: i64,
```

Comment:
Can QTUM Electrum return negative value here? Could you share an example please as I'm curious :slightly_smiling_face:

Reply:
Please read the comment above :)